From aaaff18fbbf67f88ca332d232003723865bdd813 Mon Sep 17 00:00:00 2001
From: Murat Kozan <muratkozan350@gmail.com>
Date: Sat, 8 Apr 2023 21:35:44 +0300
Subject: [PATCH] services: LightsService: Mute an annoying error message.

E LightsService: Light requested not available on this device. 2

If you google for the error you will find many discussion threads or logcats
that show this error. Since the error only describes that the device does
not support this function, the error can be classified as harmless. In order
not to confuse the user, we will now hide the logging of this error.
---
 services/core/jni/com_android_server_lights_LightsService.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/services/core/jni/com_android_server_lights_LightsService.cpp b/services/core/jni/com_android_server_lights_LightsService.cpp
index 35d8219651d74..c12f205f7d835 100644
--- a/services/core/jni/com_android_server_lights_LightsService.cpp
+++ b/services/core/jni/com_android_server_lights_LightsService.cpp
@@ -112,7 +112,7 @@ static void processReturn(
         case Status::SUCCESS:
             break;
         case Status::LIGHT_NOT_SUPPORTED:
-            ALOGE("Light requested not available on this device. %d", type);
+            ALOGV("Light requested not available on this device. %d", type);
             break;
         case Status::BRIGHTNESS_NOT_SUPPORTED:
             ALOGE("Brightness parameter not supported on this device: %d",
